LibraBFT/ImplShared/Util/RWST/Syntax.agda

```diff
@@ -67,6 +67,9 @@ caseM⊎ e of f = RWST-either e (f ∘ Left) (f ∘ Right)
 caseMM_of_ : Maybe B → (Maybe B → RWST Ev Wr St A) → RWST Ev Wr St A
 caseMM m of f = RWST-maybe m (f nothing) (f ∘ just)
 
+when : Bool → RWST Ev Wr St Unit → RWST Ev Wr St Unit
+when b f = if b then f else pure unit
```

Comment:
As with `lcheck` it may be worth extending this to anything that has a `toBool` instance.

Reply:
DONE

Reply:
@haroldcarr I think `when` should use `ifM`, and be made to work for any type with a `ToBool` instance.

Reply:
DONE: now uses `ifM`.